docs/best_practices/naming.rst

```diff
@@ -2,52 +2,70 @@ Naming
 ======
 
 
+
 Neurodata Types
 ---------------
 
 
-As a default, name class instances with the same name as the class.
 
-Many of the Neurodata Types in NWB allow you to set their name to something other than the default name.
+.. _best_practice_object_names:
+
+Naming Conventions
+~~~~~~~~~~~~~~~~~~
+
+As a default, name class instances with the same name as the class.  *E.g.*, if your :nwb-schema:ref:`sec-NWBFile` has
+only one :nwb-schema:ref:`sec-ElectricalSeries` for the raw acquisition, then simply name it ``ElectricalSeries``.
+
+Many of the Neurodata Types in NWB allow you to set their name to something other than the default name, and this should
+be done any time there are multiple types of the same class. *E.g.*, if you have several
+:nwb-schema:ref:`sec-ElectricalSeries` for different segments or devices, then add some distinguishing information to
+each name: ``ElectricalSeries1`` vs. ``ElectricalSeries2``, or ``ElectricalSeriesSegment1`` vs.
+``ElectricalSeriesSegment2``, etc. This allows multiple objects of the same type to be stored side-by-side and allows
+data writers to provide human-readable information about the contents of the ``neurodata_type``.
+
+It is recommended to use :wikipedia:`CamelCase <Camel_case>` when naming instances of schematic classes.
+
 
-This allows multiple objects of the same type to be stored side-by-side and allows data writers to provide
-human-readable information about the contents of the neurodata_type. If appropriate, simply use the name of the
-Neurodata Type as the name of that object.
 
-For instance, if you are placing an ElectricalSeries object in /acquisition that holds voltage traces for a
-multichannel recording, consider simply naming that object "ElectricalSeries". This is the default_name for that
-object, and naming it like this will increase your chances that analysis and visualization tools will operate
-seamlessly with you data.There may be cases where you have multiple neurodata instances of the same type in the same
-Group. In this case the instances must have unique names. If they are both equally important data sources, build upon
-the class name (e.g. "ElectricalSeries_1" and "ElectricalSeries_2"). If one of the instances is an extra of less
-importance, name that one something different (e.g. "ElectricalSeries" and "ElectricalSeries_extra_electrode").
```

Comment:
I really prefer this over what you changed it to.

Reply:
With the ADA, for example, it would be "ElectricalSeries_Extra_Electrode"

Reply:
changing to CamelCase is fine.

Reply:
Rephrased section to align more closely to original text.